lib/api/src/modules/channel.ts

```diff
@@ -0,0 +1,39 @@
+import deprecate from 'util-deprecate';
+import { STORY_CHANGED } from '@storybook/core-events';
+import { Module } from '../index';
+
+type CallBack = (...args: any[]) => any;
+
+export interface SubAPI {
```

Comment:
Worth adding generic types for all those `any` ?

Reply:
I'm not sure i understand this one.

Reply:
not sure if that's what @kroeder is suggesting but it'd be nice to have an option for strongly typed events so the callback can be typed.

Reply:
Sounds cool, shall we deal with that in a follow up PR?

Reply:
Definitely outside the scope of this PR